gradle/libs.versions.toml

```diff
@@ -49,6 +49,7 @@ androidx-exifinterface = { group = "androidx.exifinterface", name = "exifinterfa
 androidx-lifecycle-viewmodel-compose = { group = "androidx.lifecycle", name = "lifecycle-viewmodel-compose", version.ref = "androidx-lifecycleViewmodelCompose" }
 paging-common = { module = "app.cash.paging:paging-common", version.ref = "multiplatform-paging" }
 paging-compose-common = { module = "app.cash.paging:paging-compose-common", version.ref = "multiplatform-paging" }
+annotation = { module = "androidx.annotation:annotation", version = "1.7.1" }
```

Comment:
Can you write specific version reference like above case?

Reply:
I usually make a version variable if there are several places it's used, but yeah, sure, fixed.